cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Consider adding test cases to verify the functionality of your models and views. This will help ensure that your application behaves as expected.